usaspending_api/search/v2/views/spending_by_award_count.py

```diff
@@ -18,12 +18,10 @@
     get_generic_filters_message,
 )
 from usaspending_api.common.query_with_filters import QueryWithFilters
+from usaspending_api.search.filters.elasticsearch.filter import _QueryType
```

Comment:
Really small cleanup, can you keep this in alphabetical order for the usaspending modules?